docs/flask_config.md

```diff
@@ -20,3 +20,40 @@ def custom_route():
 Amundsen has two features that leverage the custom mail client -- the feedback tool and notifications. For these features a custom implementation of [base_mail_client](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/base/base_mail_client.py) must be mapped to the `MAIL_CLIENT` configuration variable.
 
 To fully enable these features in the UI, the application configuration variables for these features must also be set to true. Please see this [entry](application_config.md#mail-client-features) in our application configuration doc for further information.
+
+
+## Programmatic Descriptions
+Amundsen supports configuring other mark down supported non-editable description boxes on the table page.
+This can be useful if you have multiple writers which want to write different pieces of information to amundsen
+that are either very company specific and thus would never be directly integrated into amundsen or require long form text
+to properly convey the information.
+
+What are some more specific examples of what could be used for this?
+- You have an existing process that generates quality reports for a dataset that you want to embed in the table page.
+- You have a process that detects pii information (also adding the appropriate tag/badge) but also generates a simple
+report to provide context.
+- You have extended table information that is applicable to your datastore which you want to scrape and provide in the
+table page
+
+Programmatic Descriptions are referred to by a "description source" which is a unique identifier.
+You can then configure the descriptions to have a display title and order in the config.py file like so:
+```    
+PROGRAMMATIC_DISPLAY = {
+           "s3_crawler": {
+               "display_order": 0,
+               "display_title": "S3"
+           },
+           "quality_service": {
+               "display_order": 1,
+               "display_title": "Quality"
+           },
+           "doesnt_exist": {
+               "display_order": 2,
+               "display_title": "Nobody Cares"
+           }
+       }
+```
+description sources not mentioned in the configuration will be alphabetically placed at the end of the above list.
```

Comment:
```suggestion
description sources not mentioned in the configuration will be alphabetically placed at the end of the above list. If `PROGRAMMATIC_DISPLAY` is left at `None` all added fields are still showing up, so that display is entirely dynamically data-driven without configuration. Meaning configuration merely adds the (nice) benefit of setting display order.
```